_data/méli-mélo.json

```
@@ -98,13 +94,16 @@
 			"nom": "2024-04-stepsquiz",
 			"mainpage": "index.html"
 		},
-	{
+		{
+			"nom": "2024-10-datatable-utilities",
+			"mainpage": "index.html"
```

la page principale du projet est:

```suggestion
			"mainpage": "index-en.html"
```

---

Done